app/base/handlers.js

```diff
@@ -29,6 +29,9 @@ exports.home = (req, res, next) => {
 
         res.render('base/home.html', {
           tools,
+          availableTools: [
+            { tool: 'rstudio', label: 'Rstudio' },
```

Comment:
Not a biggie obviously but correct capitalisation is [RStudio](https://www.rstudio.com), missed my annoying comments :P?